internal/diagnostics/server.go

```diff
@@ -160,11 +189,11 @@ func installProfilingHandlers(mux *http.ServeMux) {
 	mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
 }
 
-// installDumpHandlers adds the config dump webservice to the given mux.
-func (s *Server) installDumpHandlers(mux *http.ServeMux) {
+// installConfigDebugHandlers adds the config dump webservice to the given mux.
+func (s *Server) installConfigDebugHandlers(mux *http.ServeMux) {
 	mux.HandleFunc("/debug/config/successful", s.handleLastValidConfig)
 	mux.HandleFunc("/debug/config/failed", s.handleLastFailedConfig)
-	mux.HandleFunc("/debug/config/problems", s.handleLastFailedProblemObjects)
+	mux.HandleFunc("/debug/config/fallback", s.handleCurrentFallback)
```

Comment:
Minor nit as I believe this is not a big issue: this is being installed regardless of the state of the fallback feature gate. This would have to be manually discovered by the users to see that this is indeed a path that can be accessed in the diagnostics server but still, it's something that they have not consciously enabled yet what KIC enabled for them.

Having said that I believe that we can leave this as is.

Reply:
I'll leave it as it is for now 👍